load_default_pseudomonas_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Script to load default Pseudomonas data (in the files included in this
+# repository) into the adage-server database via management commands.
```

Comment:
There is a `fab` task that does all of this already. Are you running it this way because you're thinking we'll eliminate our dependency on `fabric` once we've got Docker working?

Reply:
I think so - I think it is nicer to have these commands here so we do not have to rely on fabric. If I remember correctly, the fabric project was having issues with migrating to Python 3, and as a plus, we do not have to keep maintaining/thinking about our fabric code. Also, as a plus, it is nice to have these commands separated so that somebody who is not using fabric can apply all of these management commands (and also read them in a format that is more reader-friendly than the fabric format). As an example, I was thinking of when we were spinning up an instance with @jaclyn-taroni and wanted to know what exactly all the management commands to populate the database were.

Reply:
I am good with that plan. `fabric` is essentially just a Python-friendly way of scripting remote ssh, and that would be largely redundant for a Docker deployment, I think.